src/schema/schema_worker.c

```diff
@@ -68,29 +68,57 @@ __schema_tiered_worker(WT_SESSION_IMPL *session, const char *uri,
 
 /*
  * __schema_layered_worker_verify --
- *     Run a schema worker operation (which is verification) on the stable table; ingest is not
- *     supported currently.
+ *     Run a schema worker operation (which is verification) on the layered table.
  *
- * FIXME-WT-15047: Implement ingest table verification (and update the function description)
+ * FIXME-WT-15047: Implement ingest table verification on followers.
  */
 static int
 __schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
   int (*file_func)(WT_SESSION_IMPL *, const char *[]),
   int (*name_func)(WT_SESSION_IMPL *, const char *, bool *), const char *cfg[], uint32_t open_flags)
 {
+    WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
 
+    conn = S2C(session);
+
     WT_RET(__wt_session_get_dhandle(session, uri, NULL, NULL, open_flags));
     WT_LAYERED_TABLE *layered = (WT_LAYERED_TABLE *)session->dhandle;
     const char *stable_uri = layered->stable_uri;
+    const char *ingest_uri = layered->ingest_uri;
 
     WT_ASSERT(session, stable_uri != NULL);
+    WT_ASSERT(session, ingest_uri != NULL);
     WT_ASSERT(session, file_func == __wt_verify);
 
+    /*
+     * Verifying stable tables of layered tables uses the existing verify logic.
+     * Ingest tables, however, require special handling:
+     * - On leader: ingest must always be empty/no-op.
+     * - On followers: ingest contains recent oplog updates layered over the stable table.
+     */
+
+    if (conn->layered_table_manager.leader) {
+        /*
+         * On leader, if verifying ingest returns EBUSY, it means ingest is not empty (dirty
+         * content, open cursors, or checkpoints present), which is an invalid state.
```

Comment:
```suggestion
         * content, open cursors), which is an invalid state.
```

Now we return WT_ERROR if checkpoint exists

Reply:
fixed alongside other issues in 60a7dc5fad1796f5f3693636e096c076f0132af9